doc/adrs/XXX_support_representation_details_view.adoc

```diff
@@ -0,0 +1,55 @@
+= ADR-XXX - Add Support for Representation Details View
+
+== Context
+
+Currently, when a user selects a representation, nothing is displayed in the details section.
+Representations already have some metadata (id, name, targetObjectId, kind).
+But users want more information to distinguish their different representations.
+
+== Decision
+
+To address this need, we will enrich the metadata of representations by adding additional information such as a description and an icon.
+These details will be displayed in the details section when a representation is selected, and should be editable by end user.
```

Comment:
Let's be a bit more specific on what can be displayed / edited by the end user. Unless I misunderstood something:

- The label and description are edited by the end user thanks to the details view
- The icon is just displayed in the explorer / tab of a representation

Is the icon supposed to appear in the details view at all?